e2e/pages/adf/process-cloud/processListCloudComponent.ts

```diff
@@ -44,4 +59,73 @@ export class ProcessListCloudComponent {
         return this.dataTable.getAllRowsColumnValues(column);
     }
 
+    getNoOfCoulmns() {
+        return this.noOfColumns.count();
+    }
+
+    checkCloumnIdIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.id);
+        return this;
+    }
+
+    checkCloumnNameIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.name);
+        return this;
+    }
+
+    checkCloumnStatusIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.status);
+        return this;
+    }
+
+    checkCloumnStartDateIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.startDate);
+        return this;
+    }
+
+    checkCloumnAppNameIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.appName);
+        return this;
+    }
+
+    checkCloumnBusinessKeyIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.businessKey);
+        return this;
+    }
+
+    checkCloumnDescriptionIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.description);
+        return this;
+    }
+
+    checkCloumnInitiatorIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.initiator);
+        return this;
+    }
+
+    checkCloumnLastModifiedIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.lastModified);
+        return this;
+    }
+
+    checkCloumnProcessNameIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.processName);
+        return this;
+    }
+
+    checkCloumnProcessIdIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.processId);
+        return this;
+    }
+
+    checkCloumnProcessDefinitionIdIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.processDefinitionId);
+        return this;
+    }
+
+    checkCloumnProcessDefinitionKeyIsDisplayed() {
+        Util.waitUntilElementIsVisible(this.processDefinitionKey);
+        return this;
+    }
+
```

Comment:
Instead of creating a method for every column, I think it would be more relevant to create only one method : checkColumIsDisplayed(column)

Reply:
done